### DISCUSSION TITLE
eliminación de cuenta y accesibilidad a github education

### DISCUSSION THREAD

**Discussion by GARGAS17** [community/community#134136] [GitHub Education] [answered]
### Select Topic Area

General

### GitHub Feature Area

GitHub Education

### Body

hola, perdí el acceso a la cuenta de MALEKIN17 porque perdí la contraseña por el método de 2 factores, por eso desvinculo la cuenta, de hecho, si pueden eliminarla lo agradecería, además quisiera poder darle el beneficio de GitHub education a mi nueva cuenta GARGAS17 , intente iniciar la solicitud pero no me dejo, que puedo hacer en ese caso? espero su pronta respuesta Muchas gracias.

**Comment by queenofcorgis** [comment]
Hi @GARGAS17 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

_Please note: . You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._

**Comment by davevad93** [comment, marked as answer]
Hi @GARGAS17 ,

If you have lost access to the account which holds your GitHub Education benefits, unfortunately it's not possible to transfer them from one account to another. I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Two-factor authentication recovery`, and explain your issue; perhaps you can regain access to your old account so you don't lose your benefits.
